randovania/games/am2r/presets/starter_preset.rdvpreset

```diff
@@ -3,7 +3,7 @@
     "base_preset_uuid": null,
     "name": "Starter Preset",
     "uuid": "b65bf9d8-f962-4f9a-b28f-b46c03ef3d76",
-    "description": "Basic preset.",
+    "description": "This preset is aimed at those new to AM2R or its Randovania implementation. No tricks will be required to complete the seed. \n \n Noteable changes include required Launchers, new pipes between late game areas, and skippable cutscenes. Minor fixes to the base game as well as some softlock preventions are also present.",
```

Comment:
Not quite sure how to phrase it, but would like to have something like this just to stop the "wHy do YoU haVe REquiRED lAUNCHers iN thE starTEr prEsEt" questions.
```suggestion
    "description": "This preset is aimed at those new to AM2R or its Randovania implementation. No tricks will be required to complete the seed. \n \n Noteable changes to improve the gameplay experience include required Launchers, new pipes between late game areas, and skippable cutscenes. Minor fixes to the base game as well as some softlock preventions are also present.",
```

Reply:
Oh yeah, it also should say "skippable gameplay cutscenes"

Reply:
Will correct.
Sidenote: I find it weird it says "skippable" since they aren't "-able" they are just "skipped"

Reply:
PRs welcome :)